tests/callbacks/test_speed_monitor.py

```diff
@@ -31,6 +45,12 @@ def test_speed_monitor():
     wall_clock_total_calls = len(in_memory_logger.data['wall_clock/total'])
     throughput_step_calls = len(in_memory_logger.data['samples/step'])
     throughput_epoch_calls = len(in_memory_logger.data['samples/epoch'])
+    _assert_no_negative_values(in_memory_logger.data['wall_clock/train'])
```

Comment:
Did this test actually create negative values before this PR?

Reply:
I'm not sure. I never traced down the cause of the negative values.